docs/src/app/components/pages/components/app-bar.jsx

```diff
@@ -81,7 +81,7 @@ export default class AppBarPage extends React.Component {
           },
           {
             name: 'zDepth',
-            type: 'number',
+            type: 'oneOf [0,1,2,3,4,5]',
```

Comment:
i didnt get that - PropType of the component is wrong? anything that could be changed now?